index.html

```diff
@@ -1,16 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
 
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
-
-<body>
-    <!-- Add your markup here -->
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Responsive Cake webpage</title>
+        <link rel="stylesheet" href="css/style.css"
+    </head>
 
+    <body>
+        <header class="header">
+            <div class="logo-contact">
+                <span class="logo">
+                    <img src="https://lh3.googleusercontent.com/proxy/BhNZ79ahYgRQMqiAm86fWvnJ0jSJWWU_18X2V1cfHXopFFu2ZixnwSJppz7HIq9zjwwrSXHJU_EA8qnklrT2JGHvh-IQK5A" alt="cake animated image"/>
```

Comment:
I would try to avoid using external urls for images as it can often lead to broken links, which is what happens for me when I check this out locally. I get a 403 error which means access is forbidden. Anyone without the relevant permissions to see the image will be left with a broken link.